src/stores/MorseStore.js

```diff
@@ -29,20 +32,63 @@ class MorseStore extends SettingsSaver {
           speed: this.speed,
           frequency: this.frequency,
           delay: this.delay,
-          maxRepeats: this.maxRepeats
+          maxRepeats: this.maxRepeats,
+          activeDictionarySize: this.activeDictionarySize,
+          types: this.types
         };
       }
     });
 
+    this.setupActiveDictionary();
     this.setupSettings("MorsePlayer", noDebounce);
   }
 
+  includeCount = () => {
+    var count = 0;
+
+    Object.entries(this.types).forEach(([key, value]) => {
+      if (value) {
+        count++;
+      }
+    });
+
+    return count;
+  };
+
+  setupActiveDictionary = () => {
```

Comment:
Is there any circumstance in which one would change this.types without wanting to call setupActiveDictionary? If not, I suggest calling it from setType and setTypes, so that you don't need to remember to do it everywhere else.

Reply:
Not that I can think of. I will make the suggested change.